tests/openvino/test_modeling.py

```diff
@@ -600,7 +616,7 @@ def test_default_filling_attention_mask(self):
         attention_mask = tokens.pop("attention_mask")
         outs_without_attn_mask = model_with_cache(**tokens)
         self.assertTrue(torch.allclose(outs.logits, outs_without_attn_mask.logits))
-        input_ids = torch.argmax(outs.logits, dim=2)
+        input_ids = torch.argmax(outs.logits[:, -1:, :], dim=2)
```

Comment:
why is this needed ?

Reply:
logits after first iteration contains initial seq len, but we need only the last token, so for reducing calculations I filter only the last new token